src/extension.ts

```diff
@@ -174,6 +174,8 @@ export class ExtensionManager implements vscode.Disposable {
             }
             await this.initActiveProject();
         }
+        rollbar.takePromise('Post-folder-open', { folder: this.getActiveProject()?.folderPath }, this.postWorkspaceOpen(this.getActiveProject()));
```

Comment:
It doesn't seem like this one is needed.  `postUpdateActiveProject` is called before we get here so `postWorkspaceOpen` will have already been called.